cmd/serverless/main.go

```diff
@@ -230,6 +231,14 @@ func runAgent(ctx context.Context, stopCh chan struct{}) (err error) {
 
 	// read configuration from the environment vars
 	// --------------------------------------------
+	svc := sts.New(session.New())
+	functionARN, err := aws.FetchFunctionARNFromEnv(svc)
```

Comment:
I think it might be cleaner for `FetchFunctionARNFromEnv()` to just take an account ID instead of an sts session. Right now it is just passing that through to `fetchAccountID()` anyway. This would let us test `FetchFunctionARNFromEnv()` without mocking STS.

Reply:
Good point, I'll move it out.